frontend/config/webpack.prod.babel.js

```diff
@@ -0,0 +1,38 @@
+var path = require('path');
+var webpack = require('webpack');
+
+module.exports = {
```

Comment:
Yeah I know, this is a first implementation just to get a feel for it. :grin:

I'll probably do it how we did it for `react-boilerplate` with a `webpack.base.babel.js`: https://github.com/mxstbr/react-boilerplate/tree/master/internals/webpack (the `if/else` in a single file gets unmaintainable when config gets more complex)